### PR TITLE
[AIRFLOW-2114] Add AIRFLOW_HOME to PythonVirtualenvOperator

### PR DESCRIPTION
**Pull request by john-bodley**
Make sure you have checked _all_ steps below.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title.
    - https://issues.apache.org/jira/browse/AIRFLOW-2114

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:

This PR passes the `AIRFLOW_HOME` environment variable to the subprocess which is required in order for the connection information to be available within the subprocess. The logic is derived from the [BashOperator](https://github.com/apache/incubator-airflow/blob/master/airflow/operators/bash_operator.py#L81).

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Existing unit test converage.

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: airflow/operators/python_operator.py [hunk 1]
```diff
@@ -29,6 +30,9 @@
 from textwrap import dedent
 
 
+AIRFLOW_HOME_VAR = 'AIRFLOW_HOME'
```
Comment (artwr) [inline airflow/operators/python_operator.py:33]: Shouldn't this be in the conf? and imported at the same time? If this changes for any reason, people won't know to look for it inside here

**Review by artwr — COMMENTED** [review]
Small comment but it looks ok.
I am not sure about the need for a constant here versus storing it in the config.

File: airflow/operators/python_operator.py (hunk 1 above)
Comment (john-bodley) [reply airflow/operators/python_operator.py:33]: @artwr the `BashOperator` does something similar.

Comment (aoen) [reply airflow/operators/python_operator.py:33]: the env var should be de-dupped at least.

Comment (john-bodley) [reply airflow/operators/python_operator.py:33]: @aoen it seems that the subprocess command doesn't support prepending environment variables in the same manner of the [BashOperator](https://github.com/apache/incubator-airflow/blob/master/airflow/operators/bash_operator.py#L84), also it's not apparent what shell is being run. From a Google search the approach I've taken here with regards to augmenting the environment seems to be consistent with others.

**Comment by john-bodley** [comment]
Closing this after discussing with @aoen offline.

**Pull request closed without merging**